src/test/java/org/springframework/samples/petclinic/service/AbstractClinicServiceTests.java

```diff
@@ -366,12 +368,29 @@ void shouldDeleteVisitsById() {
 //    }
 
     @Test
+    @Order(21)
     void shouldFindAllAppointments(){
 
         Collection<Visit> visits = this.clinicService.findAllVisits();
         assertThat(visits.size()==4);
 
     }
 
+    @Test
+    @Transactional
+    @Order(19)
+    void testCancelFunctionality() throws Exception{
```

Comment:
This should follow the same naming convention as the other tests (rename to `shouldDeleteVisitsById` and rename the existing `shouldDeleteVisitsById` to `shouldDeleteVisitsByIdIn`)

Reply:
Added requested changes in latest push.